packages/visualizations/src/components/Map/utils.js

```diff
@@ -1,19 +1,40 @@
 import chroma from 'chroma-js';
 import geoViewport from '@mapbox/geo-viewport';
 
-export const colorShapes = (geoJson, values, colorScale) => {
+export const colorShapes = (geoJson, values, colorsScale) => {
     // Key in the values is "x"
     // Key in the shapes is "key"
     // We add a color property in the JSON
+    // FIXME Do we always want to take y or accept other keys
```

Comment:
That's a good question. I always assumed we can enforce a convention where we want the data to have `x` and `y` (and maybe `label` for labels etc). I think it's reasonable, since here, worst case, you could write 2 lines of code to rename the keys before calling our own code.

Or did you have in mind a future situation where we could use more than one value in the calculations?

Reply:
It was just a reminder to check if this will be a convention in the final version, and i guess we can indeed enforce this for now, I have no future situations in mind that would require something else, so I'll remove this comment !